doc/CRD-generator.md

```diff
@@ -442,3 +442,32 @@ The CRD generator will add the additional `labels`:
 | `io.fabric8.crd.generator.annotation.Labels`                 | Additional `labels` in `metadata`                                                     |
 
 A field of type `com.fasterxml.jackson.databind.JsonNode` is encoded as an empty object with `x-kubernetes-preserve-unknown-fields: true` defined.
+
+## Experimental
+
+### Generate CRDs in parallel
+Starting from 6.4, it's possible to speed up the CRDs generation by using parallel computation.
```

Comment:
6.4.0 is already released, since this change doesn't break the API it could be introduced in a patch 6.4.x release